docs/crd.md

```diff
@@ -151,6 +154,9 @@ FlinkCluster
       * **nodeSelector** (optional): Selector which must match a node's labels for the JobManager pod 
         to be scheduled on that node.
         See [More info](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/)
+      * **tolerations** (optional): Allows the JobManager pod to run on a tainted node
+        in the cluster.
+        See [More info](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)  
```

Comment:
nit: s/More/more. Could update all the places?

Reply:
All set. Updated nodeSelector and tolerations.